app/Template/notification/task_overdue.php

```diff
@@ -29,3 +35,5 @@
         </tr>
     <?php endforeach ?>
 </table>
+
+<?= $this->render('notification/footer', array('task' => array('project_id' => $project_id))) ?>
```

Comment:
The footer is not really needed since you have added the link to the board and the task just above.

You may have overdue notifications for different projects.